pkg/kubelet/kubelet.go

```diff
@@ -840,6 +855,8 @@ func (kl *Kubelet) GetPodDir(podUID types.UID) string {
 	return kl.getPodDir(podUID)
 }
 
+// getPodDir returns the foll path to the per-pod directory for the pod with
+// the given UID.
 func (kl *Kubelet) getPodDir(podUID types.UID) string {
```

Comment:
I don't know, and I do not want to add or remove code in this PR if at all possible, just change doc for now -- let's address this in a follow up.